### ISSUE TITLE
[BITV/A11Y] ActionMenu accessibility

### ISSUE THREAD

**Issue by PVince81** [nextcloud-libraries/nextcloud-vue#2502]
* [ ] Menu opening button does not need `aria-haspopup` because it is not a `role="menu"`, but a hasmenu, and is deprecated. Can be removed because we have `aria-expanded`
  * [ ] Menu should be inserted in the DOM directly behind the button itself → this would also fix the bug where in the files list on scrolling, the menu scrolls away.
  * [ ] `aria-label` property is not needed as there is a text label already, can be removed.
  * [ ] Focus should initially stay on the toggle button, not move into first entry of the list
  * [ ] Focus should not loop through the menu but after last entry, close the menu and move to next
  * [ ] Same with Shift-Tab, should move back to toggle button after first entry
  * [ ] Focus needs to get border

**Comment by JuliaKirschenheuter** [comment]
Menu-related requirements are described here https://github.com/nextcloud/server/issues/37099
Menu-related focus trap is described in other known issues
Focus border was discussed in other issues, not needed to be applied

Moving issue to "descoped" for now

**Comment by JuliaKirschenheuter** [comment]
closing as done
